### ISSUE TITLE
Double reading problem in Windows

### ISSUE THREAD

**Issue by stevefan1999-personal** [zyansheep/rustyline-async#12]
Minimal reproducible program later, but here's what the symptoms looks like:
```
> tteesstt  -- I typed test
> tteesstt> -- Then I pressed enter
>
> >
>
>
```

**Comment by zyansheep** [comment]
Seems to be a crossterm issue lol: crossterm-rs/crossterm/issues/772
The [fix](https://github.com/crossterm-rs/crossterm/issues/772#issuecomment-1493373552) seems relatively easy but I don't use Windows so I couldn't test it... a PR would be much appreciated :)

I think this is a recent crossterm bug, so maybe downgrading the version would work as a temporary fix

**Comment by zyansheep** [comment]
Should be fixed by #14
